documentation/dev/docker-compose.rst

```diff
@@ -0,0 +1,80 @@
+.. _docker-compose:
+
+Running a complete stack with docker-compose
+=============================================
+
+Installing FlexMeasures and dependencies is some work, and can have unexpected hurdles, e.g. depending on the operating system. So, running the whole stack via `Docker compose <https://docs.docker.com/compose/>`_ can be a very useful option.
+
+For this, we assume you are in the directory housing ``docker-compose.yml``.
+
+
+Build the compose stack
+------------------------
+
+Run this:
+
+.. code-block:: bash
+
+    docker-compose build
```

Comment:
First time I ran this I got:

```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```
Deleting the version from `docker-compose.yml`, then running again gave:
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services: 'dev-db'
Unsupported config option for volumes: 'flexmeasures-instance'
```

Reply:
Why  would you simply delete the version?
What is your docker-compose version? Maybe we have a minimal requirement.

Reply:
It seems to be the docker version which is relevant: https://docs.docker.com/compose/compose-file/compose-versioning/

I can choose a compse version a little lower. But let me know which you have ...

Reply:
I have 19.03.13

Reply:
I found a list of upgrades in between versions, and I'm using a feature from Docker Compose 3.4. 

Still interested what docker version you have, but for now, I'm setting the version to 3.4. 3.8 is also not very recent (1.5 years old I think) but this should be safer for older Docker installations.

Let me know if that works for you.

Reply:
`docker-compose --version` gave 1.25.0 for me. It now works after I pulled your new commits.